projects/core/dsl/toolbox.py

```
@@ -0,0 +1,177 @@
+        for k, v in value.items():
+            cleaned_args[k] = str(v) \
+                if isinstance(v, pathlib.Path) \
+                   else v
+        return cleaned_args
+
+    yield "x" * 80
+    yield f"~~ {e.task_location}"
+    yield f"~~ TASK: {e.task_name}: {e.task_description}"
+    yield f"~~ ARTIFACT_DIR: {e.artifact_dir}"
+    yield f"~~ LOG_FILE: {e.artifact_dir}/task.log"
+    yield f"~~ ARGS:"
```

Ruff (0.15.9)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)